ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml

```diff
@@ -126,6 +126,16 @@ objects:
       - /usr/bin/kubectl
       - /tmp/shared/
 
+    - name: cli-tests
+      image: ${IMAGE_TESTS}
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      command:
+      - cp
+      - /usr/bin/openshift-tests
+      - /tmp/shared/
```

Comment:
if your test has a need for openshift-tests, then write a dockerfile that builds an image that includes it and use that as the image for your test.